pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -7,11 +7,11 @@
 { lib, stdenv, pkg-config, pango, perl, python3, zip
 , libjpeg, zlib, dbus, dbus-glib, bzip2, xorg
 , freetype, fontconfig, file, nspr, nss
-, yasm, libGLU, libGL, sqlite, unzip, makeWrapper
+, libGLU, libGL, sqlite, unzip, makeWrapper
 , hunspell, libevent, libstartup_notification
 , libvpx
-, icu70, libpng, glib, pciutils
-, autoconf213, which, gnused, rustPackages, rustPlatform
```

Comment:
Can we carry this commit easily over to spidermonkey and palemoon? If so we could drop autoconf213.

Reply:
Spidermonkey 78 no, 91 maybe.